app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="w-screen h-screen flex">
+
+  <div class="max-w-[600px] max-h-[750px] w-full h-full m-auto rounded rounded-lg shadow-[0_0_20px_0_rgba(0,0,0,0.15)] flex flex-wrap">
```

Comment:
let's [change this px measure to rem](https://nekocalc.com/px-to-rem-converter), which is more mobile-friendly